index.html

```diff
@@ -1,65 +1,207 @@
+                    <div class="project-info">
+                        <div class="company">CANOPY</div>
+                        <div class="dot"></div>
+                        <div class="position">Back End Dev</div>
+                        <div class="dot"></div>
+                        <div class="year">2015</div>
+                    </div>
+                    <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="lang">HTML</li>
+                        <li class="lang">CSS</li>
+                        <li class="lang">Javascript</li>
+                    </ul>
+                    <div class="see-project">
+                        See project
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="about-section">
+            <div class="about-info">
+                <p class="title">About Myself</p>
+                <p class="content">
+                    Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
                 <p class="connect">LET'S CONNECT</p>
-                
+                        
                 <ul class="social-icons">
                     <li class="icon">
-                        <img src="./images/twitter.png">
+                        <img src="./images/github.png">
                     </li>
                     <li class="icon">
                         <img src="./images/linkedin.png">
                     </li>
-                    <li class="icon medium">
-                        <img src="./images/medium.png">
-                    </li>
                     <li class="icon">
-                        <img src="./images/github.png">
+                        <img src="./images/taobao.png">
                     </li>
                     <li class="icon">
-                        <img src="./images/taobao.png">
+                        <img src="./images/twitter.png">
+                    </li>
+                    <li class="icon medium">
+                        <img src="./images/medium.png">
                     </li>
                 </ul>
+        
+                <div class="see-project get-my-resume">
+                    Get my resume
+                </div>
             </div>
+            
+            <ul class="about-list">
+                <li class="about-item">Languages<img src="./images/Icon - Chevron - Down - Larger.png"></li>
+                <div class="about-lang"><img src="./images/Ellipse 1.png">Javascript</div>
+                <div class="about-lang"><img src="./images/Ellipse 1(1).png">HTML</div>
+                <div class="about-lang"><img src="./images/Ellipse 1(2).png">CSS</div>
```

Comment:
- [ ] Please 🙏🏻  use the `<li>` tag over `div` to enhance the professionalism and semantic correctness of your code, place each social media icon link within an individual **li** tag. This approach not only improves the structure of your *HTML* but also benefits **visually-impaired** individuals, ensuring a better user experience. Avoid using the `<div>` tag solely for decorative purposes, as it lacks semantic meaning.

   - Once you make the necessary changes, remember to add appropriate styles to your CSS file. Implementing 
     **flexbox** can help you properly align the items within the **ul** container. Utilizing `Flexbox` provides a flexible 
     and responsive layout, ensuring consistent presentation across different screen sizes and devices.

   - It is crucial to always close your tags with the correct tag name. This practice becomes even more significant as your code scales or when you add more features in the future. By diligently closing your tags, you avoid 
     potential errors that may arise due to small oversights. Maintaining proper tag closure ensures code readability, makes debugging easier, and helps maintain the integrity of your markup.